crates/vm/levm/src/errors.rs

```diff
@@ -218,16 +218,6 @@ pub enum DatabaseError {
     Custom(String),
 }
 
-#[derive(Debug, Clone)]
-/// Note: "Halt" does not mean "Error during execution" it simply
-/// means that the execution stopped. It's not called "Stop" because
-/// "Stop" is an Opcode
-pub enum OpcodeResult {
-    Continue { pc_increment: usize },
-    SetPc { new_pc: usize },
-    Halt,
-}
```

Comment:
Can we keep this as an enum? Like:

```rust
pub enum OpcodeResult {
    Continue,
    Halt,
}
```

It should yield the same performance but being more readable.

Reply:
Fixed.